src/commands/projects.ts

```diff
@@ -108,15 +111,29 @@ export const builder = (argv: yargs.Argv) => {
       'Update a project',
       (yargs) =>
         yargs.options({
-          name: {
-            describe: projectCreateRequest['project.name'].description,
-            type: 'string',
+          'block-vpc-connections': {
+            describe:
+              projectUpdateRequest['project.settings.block_vpc_connections']
+                .description +
+              ' Use --block-vpc-connections=false to set the value to false.',
```

Comment:
there is also an option to prefix with `--no-` :) But I don't think we promote this anywhere.